src/sensor/settings.py

```diff
@@ -8,6 +8,10 @@
 For the full list of settings and their values, see
 https://docs.djangoproject.com/en/1.11/ref/settings/
 
+!!!!!!NOTE!!!!!: This file is replaced when scos-sensor runs in docker. migration_settings.py is used when migrations are
```

Comment:
When is this file used, if not when running in Docker? Is this working towards the goal of being able to run actions without scos-sensor?